src/solver/conv_winoRxS_fused.cpp

```diff
@@ -190,9 +190,9 @@ bool ConvBinWinogradRxSf2x3g1Fused::IsApplicable(const FusionContext& context,
     const auto OH = conv_ctx.problem.conv_problem.GetOutHeight();
     const auto OW = conv_ctx.problem.conv_problem.GetOutWidth();
 
-    return IsWinogradV30Supported<2, 3>(name, conv_ctx.problem)
-               ? IsShaderContraintsMetV30(conv_ctx.problem, R, S, C, K, H, W, OH, OW, N)
-               : IsShaderContraintsMetV21(conv_ctx.problem, R, S, C, K, H, W, OH, OW, N);
+    return IsWinogradV21Preferred<2, 3>(name, conv_ctx.problem)
+               ? IsShaderContraintsMetV21(conv_ctx.problem, R, S, C, K, H, W, OH, OW, N)
+               : IsShaderContraintsMetV30(conv_ctx.problem, R, S, C, K, H, W, OH, OW, N);
```

Comment:
Typo in `IsShaderContraints*`. Should be `IsShaderConstraints*`

Reply:
Thank you, fixed in [106c8a8](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1968/commits/106c8a833e775ae048a11b7f7d6dba5a54ad1991)